src/pages/patterns/buttons.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: Buttons
+description: >
+  Know why they're pushing your buttons.
```

Comment:
Please apply a curly quote here as the gatsby plugin doesn't hit this portion of MDX.

Reply:
Rather than have us manually enter these, I've added a customization to our page creation logic that runs all frontmatter title and description fields through https://smartquotes.js.org/

This also corrected some rogue quotes in our component pages.

Buttons component before:
![before](https://user-images.githubusercontent.com/4030377/107267281-87252b80-69fb-11eb-863a-1a2c882865c8.png)

Buttons component after:
![after](https://user-images.githubusercontent.com/4030377/107267304-8e4c3980-69fb-11eb-8308-c59704007669.png)

Reply:
Awesome :tada: